operator/v1/types_kubeapiserver.go

```diff
@@ -35,6 +35,23 @@ type KubeAPIServer struct {
 
 type KubeAPIServerSpec struct {
 	StaticPodOperatorSpec `json:",inline"`
+
+	// eventTTL specifies the amount of time that the events are stored before being deleted.
+	//
+	// The value must be parseable as a time duration value;
+	// see <https://pkg.go.dev/time#ParseDuration>.
+	//
+	// If configured, it must be a value of 1m (one minute) or greater, we only allow setting
+	// minute and hour durations (e.g. 5m or 5h).
```

Comment:
We should set a sensible maximum value for this TTL. Can you suggest a sensible maximum value for this?

Reply:
yep, made it 2x the default of 3h now